Translation corrections (#462)
07th-mod/higurashi-console-arcs: Update/omot_014.txt

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#5ec69a>詩音</color>", NULL, "<color=#5ec69a>Shion</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 6, "ps3/s14/06/210600560", 256, TRUE);
	OutputLine(NULL, "そもそもダム工事にまつわって村の掟に逆らう人が対象となるなら、４年前現場監督を殺して行方不明になった作業員は？」",

> 			NULL, " If the curse targeted people who defied the villages laws and were involved with the dam project... then what about the construction worker who killed the on-site construction manager and vanished?\"", Line_Normal);

Reviewer: ```suggestion
			NULL, " If the curse targeted people who defied the village laws and were involved with the dam project... then what about the construction worker who killed the on-site construction manager and vanished?\"", Line_Normal);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#956f6e>圭一</color>", NULL, "<color=#956f6e>Keiichi</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 1, "ps3/s14/01/210101344", 256, TRUE);
	OutputLine(NULL, "「…つまり、この事件がきっかけとなって、それ以前に起こったものも含めた一連の事件が、不可思議な現象に変わったんだと思う」",

> 			NULL, "\"...In other words, that incident changed everyone's perception of what happened the past two years as a mysterious phenomenon, rather than a series of incidents.\"", Line_Normal);

Reviewer: ```suggestion
			NULL, "\"...In other words, that incident changed everyone's perception of what happened the past two years, turning the series of incidents into a mysterious phenomenon.\"", Line_Normal);
```
